views/form.blade.php

```diff
@@ -14,9 +14,12 @@
             @if(!$tabObj->isEmpty())
                 @include('admin::form.tab', compact('tabObj'))
             @else
-                <div class="fields-group">
+                <div class="fields-group row" style=" display: flex;flex-wrap: wrap;">
```

Comment:
You need it when you don't have rows for each 12 cols. 
http://stackoverflow.com/a/42607629/936651